python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_sensors.py

```diff
@@ -397,6 +397,33 @@
 }
 """
 
+GET_TICK_DYNAMIC_PARTITIONS_REQUEST_RESULTS_QUERY = """
+query TickDynamicPartitionsRequestResultsQuery($sensorSelector: SensorSelector!) {
+  sensorOrError(sensorSelector: $sensorSelector) {
+    __typename
+    ... on PythonError {
+      message
+      stack
+    }
+    ... on Sensor {
+      id
+      sensorState {
+        id
+        ticks {
+          id
+          dynamicPartitionsRequestResults {
+            partitionsDefName
+            partitionKeys
+            skippedPartitionKeys
```

Comment:
Should we add this field to the Dry run version of this too?

https://github.com/dagster-io/dagster/blob/0de69c34b0378cb795922be270fd76b5ae6a89e6/python_modules/dagster-graphql/dagster_graphql/schema/instigation.py#L378